packages/block-library/src/social-links/edit.js

```diff
@@ -166,7 +166,7 @@ export function SocialLinksEdit( props ) {
 					) }
 				</ToolbarDropdownMenu>
 			</BlockControls>
-			<InspectorControls>
+			<InspectorControls __experimentalExposeToChildren>
```

Comment:
I didn't think too much about whether it makes sense to expose these two inspector controls present in the Social Links blocks. Let's assume it's only for demo and testing purposes. I'm happy to expose other inspector controls from different blocks as part of this PR or keep it outside of the scope.